src/components/PageElements/Elements/ReviewInlineNewResponse.tsx

```diff
@@ -0,0 +1,132 @@
+import React, { useState } from 'react'
+import { Button, Form, Radio, TextArea } from 'semantic-ui-react'
+import { useLanguageProvider } from '../../../contexts/Localisation'
+import { SummaryViewWrapperProps } from '../../../formElementPlugins/types'
+import { ReviewResponseDecision } from '../../../utils/generated/graphql'
+import { useCreateReviewResponseMutation } from '../../../utils/generated/graphql'
+import { ReviewDetails } from '../../../utils/types'
+
+interface ReviewInlineNewResponseProps {
```

Comment:
This component is mostly the same as `ReviewInlineInput`, just with a few modifications to handle creating a new response rather than updating an existing one.